ceph-build/build/setup_rpm

```diff
@@ -40,7 +40,7 @@ fi
 sed -e 's/@//g' < ceph.spec.in > $DIR/ceph.spec
 
 # Make sure we have all the rpm macros installed and at the latest version before installing the dependencies
-$SUDO yum install -y \*rpm-macros
+$SUDO yum install -y python36-devel
```

Comment:
this is a bit odd... is the comment for this line outdated now? or is python36-devel pulling the macros?

Reply:
if it is pulling the rpm-macros can we update the comment to reflect that? this totally caught me off guard

Reply:
@alfredodeza updated and repushed.